### PR TITLE
stk: update to 5.0.1

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- cmake updates

**Test Plan**
- Rebuilt lmms against this

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/s/stk/pspec_x86_64.xml [hunk 1]
```diff
@@ -19,8 +20,8 @@
 </Description>
         <PartOf>multimedia.library</PartOf>
         <Files>
-            <Path fileType="library">/usr/lib64/libstk-4.6.1.so</Path>
-            <Path fileType="library">/usr/lib64/libstk.so</Path>
+            <Path fileType="data">/lib64/libstk-5.0.0.so</Path>
+            <Path fileType="data">/lib64/libstk.so</Path>
```
Comment (EbonJaeger) [inline packages/s/stk/pspec_x86_64.xml:24]: This doesn't look correct. I think the `package.yml` `libdir` path needs to be updated for recent symlink changes.

**Review by EbonJaeger — CHANGES_REQUESTED** [review]

File: packages/s/stk/pspec_x86_64.xml (hunk 1 above)
Comment (androidnisse) [reply packages/s/stk/pspec_x86_64.xml:24]: What would be the correct path here to add?

Comment (EbonJaeger) [reply packages/s/stk/pspec_x86_64.xml:24]: We have a `%libdir%` variable macro, so I would try that.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**